functions/index.js

```diff
@@ -48,6 +48,8 @@ const { performanceChart } = require("./model/performanceChart/performanceChart"
 //         this.StoryPointNumber = storyPointNumber;
 //     }
 // }
+const { performanceChart } = require("./model/performanceChart/performanceChart")
```

Comment:
performanceChart is present twice. Fix that.
Also remove the commented code.
And add semicolon at the end of line 51 and 52